recipes/newrelic/infrastructure/ohi/rabbitmq/windows.yml

```diff
@@ -0,0 +1,63 @@
+name: rabbitmq-open-source-integration
+displayName: RabbitMQ Open Source Integration
+description: New Relic install recipe for default RabbitMQ Open Source on-host integration (via Infra-Agent)
+repository: https://github.com/newrelic/nri-rabbitmqi
+
+installTargets:
+  - type: host
+    os: windows
+    kernelArch: x86_64
+
+# keyword convention for dealing with search terms that could land someone on this instrumentation project
+keywords:
+  - Infrastructure
+  - Integration
+  - rabbitmq
+
+processMatch:
+  - erlsrv\.exe
+
+preInstall:
+  requireAtDiscovery: |
+    powershell -command '
+    try{
+      if (Get-Service "rabbitmq" -ErrorAction SilentlyContinue) {
+         exit 132
+      }
+      
+      $rabbitMQMgmtUrl = "http://localhost:15672/"
+      $response=Invoke-WebRequest -UseBasicParsing -Method Get -TimeoutSec 10 -Uri $rabbitMQmgmtUrl
+      $foundRabbitMQMgmt = $response -match "rabbitmq" -ieq "True"
+      
+      if($foundRabbitMQMgmt){
```

Comment:
@NRhzhao this is great. I think we should also check for a response code 200 in addition of checking we have something that matches the string "rabbitmq" in the response, just in case we get an error with an html content that happened to have that string in it.